make/platform/Makefile.linux64

```diff
@@ -17,3 +17,5 @@
 
 # Makefile.linux64
 # Special AR and RANLIB definitions for XMT used to be here.
+
+CXXFLAGS += -frtti
```

Comment:
not sure about that, if I don't add this, it'll fail with some dynamic functions, like isa<>()...